sdk/appcenter-storage/src/test/java/com/microsoft/appcenter/storage/StorageTest.java

```diff
@@ -311,6 +318,27 @@ public void deleteEndToEnd() {
         // TODO: assert error is null and document is null
     }
 
+    @Test
+    public void tokenClearedOnSignOut() {
+        Set<String> partitionNames = new HashSet<>();
```

Comment:
Add comments /* when */ /* Call */  /* verify */

Reply:
Well, we don't really care in Android code. 😄

Reply:
@YiMysty there are different code style conventions in Android SDK :)

Reply:
I've made these comments in the past, I guess we don't have a hard requirement for it in android.  Sorry about that @YiMysty 

Although I do think they're a good idea and if you want to add them that would be great :)